src/components/pages/room.js

```diff
@@ -30,32 +34,35 @@ export default function Room({ PageTemplate, initialState }) {
     window.location.reload();
```

Comment:
This doesn't work so nicely anymore. After each invite, you get redirected to the lessons tab, Andreas had a suggestion that we deep link the tab key in the URL. I'm not sure if you want to go for that now.

Reply:
I know we discussed that and we should definitely do it. I would not do that here though, but in a ticket that covers all such case.